frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.module.css

```diff
@@ -53,13 +55,13 @@ body {
   where the EmbedFrame is not part of the exported dom */
       .WithThemeBackground {
     background-color: var(--mb-color-bg-dashboard);
-    border-color: var(--mb-color-bg-dark);
+    border-color: var(--mb-color-border);
```

Comment:
This fixes the [wrong border issue](https://github.com/metabase/metabase/pull/46428#pullrequestreview-2216037341) reported by Alberto.